source/versioned-api/versioned-api.rst

```diff
@@ -181,20 +184,18 @@ server description MUST reflect this with an ``Unknown`` server type.
 Cursors
 ~~~~~~~
 
-The ``getMore`` command does not accept API parameters; cursors inherit their
-API parameters from the initiating command. Drivers MUST NOT include API
-parameters when sending ``getMore`` commands to the server.
-
-In contrast, the ``killCursors`` command accepts API parameters and drivers MUST
-include them as they would with all other commands.
+For ``getMore``, drivers MUST submit API parameters. If the values given do not
+match the API parameters given in the cursor-initiating command, the server will
+reply with an error.
 
 
 Transactions
 ~~~~~~~~~~~~
 
-When running commands as part of a transaction, drivers MUST NOT send API
-parameters after the initial command that includes the ``startTransaction``
-option.
+When running commands as part of a transaction, drivers MUST send API parameters
+with all commands that are part of a transaction. If the API parameters for a
```

Comment:
Mention "including commitTransaction and abortTransaction"?

Reply:
Good point, added 👍